.github/workflows/make-pr-ready-for-review.yaml

```diff
@@ -0,0 +1,36 @@
+name: PR Ready for Review on Label Removal
+
+on:
+  pull_request:
+    types:
+      - unlabeled
```

Comment:
does this mean the action won't run on PRs that have labels?

Reply:
Ah, it means it is triggered by remove label events....

Reply:
yes